frontend/src/lib/components/heroEmoji/HeroEmoji.svelte

```diff
@@ -0,0 +1,41 @@
+<script lang="ts">
+  import type {HeroEmojiProps} from './HeroEmoji.type';
+
+  type $$Props = HeroEmojiProps;
+
+  // Merge the necessary classes to possible extra classes defined in the props
+  $$restProps.class = `whitespace-nowrap truncate text-clip text-center font-emoji text-[6.5rem] leading-none ${
+    $$restProps.class ?? ''
+  }`;
+</script>
+
+<!--
+@component
+Used for large emojis acting as decorative illustrations.
+
+The content of the component is the emoji or emojis to be displayed.
+
+The content is hidden from screen readers by default, because the
+intended use is for [decorative purposes](https://www.w3.org/WAI/tutorials/images/decorative/).
+To override, add `aria-hidden="false"` to the tag and also consider
+adding an `aria-label`.
+
+To change the size of the emoji, add a `text-[size]` utility class
+in the `class` property, e.g. `text-[10rem]`.
```

Comment:
```suggestion
using the `class` attribute, e.g. `class="text-[10rem]"`.
```
or something like that.